docs/source/installation.rst

```diff
@@ -356,82 +395,135 @@ following contents:
 
 .. code-block:: bash
 
   sbatch run.sh
 
+The benefit of non-interactive job submissions is that your job will be executed automatically,
+when the requested computational resources are available and that you can submit and queue multiple
+jobs.
 
-Install on Baskerville (singularity)
-------------------------------------
-
-In order to install parakeet on the baskerville tier 2 supercomputer with
-singularity, start an interactive job as follows (you will need to know your
-account number and qos to do this):
+To run parakeet interactively (https://docs.baskerville.ac.uk/interactive-jobs/), execute:
 
 .. code-block:: bash
-  
-  salloc --account=${ACCOUNT} --qos=${QOS} --gpus=1 --time=1:0:0
-  srun --pty bash -i
 
-Now run the following commands:
+  srun --account=${ACCOUNT} --qos=${QOS} --gpus=1 --time=${TIME} \
+    --export=USER,HOME,PATH,TERM --pty /bin/bash
+
+When the interactive job starts (you may need to wait until the requested computational resources are
+available), execute the following commands to install parakeet:
 
 .. code-block:: bash
 
   # Load required modules
   module purge
   module load baskerville
-  module load bask-singularity-conf/live
+  module load CUDA/11.4
+  module load FFTW
+  module load Python/3
 
-  # Install package
-  singularity build parakeet.sif docker://ghcr.io/rosalindfranklininstitute/parakeet:master
+  # Activate environment
+  source env/bin/activate
+
+  # Parakeet commands
+  parakeet.config.new -c config.yaml
+  parakeet.sample.new -c config.yaml
+  parakeet.sample.add_molecules -c config.yaml
+  parakeet.simulate.exit_wave -c config.yaml
+  parakeet.simulate.optics -c config.yaml
+  parakeet.simulate.image -c config.yaml
+
+
+Install on Baskerville (apptainer)
+------------------------------------
+
+Alternatively, you may run parakeet on the baskerville tier 2 supercomputer with
+apptainer (https://docs.baskerville.ac.uk/containerisation/).
 
-Once you are happy, log out of the interactive node. To run parakeet on
-baskerville write a script called run.sh with the following contents:
+To run parakeet on Baskerville non-interactively (https://docs.baskerville.ac.uk/jobs/)
+you need write a script called run.sh with the following contents (you will need to know
+your account-project code, qos and how much time do you expect to take for your script to finish):
 
 .. code-block:: bash
 
   #!/bin/bash
   #SBATCH --account=$ACCOUNT
   #SBATCH --qos=$QOS
   #SBATCH --gpus=1
+  #SBATCH --time=$TIME
 
   # Load required modules
   module purge
   module load baskerville
-  module load bask-singularity-conf/live
+  module load CUDA/11.4
+  module load FFTW
+  module load Python/3
+
+  export APPTAINER_CACHEDIR=/path/to/cache/directory/within/your/project/directory
 
   function parakeet {
-    singularity run --nv parakeet.sif parakeet $@
+    apptainer run --nv quay.io/rosalindfranklininstitute/parakeet:latest $@
   }
 
   # Parakeet commands
-  parakeet run -c config.yaml
```

Comment:
I don't think this change works. The previous command worked because "run -c config.yaml" is passed as the arguments to aptainer run. The "parakeet.*" commands won't work here I think